test/artefact_test.rb

```diff
@@ -21,16 +21,6 @@
       assert_equal @tag1, Slimmer::Artefact.new(@data).primary_section
     end
 
-    it "should ignore other tag types" do
-      @data["tags"].unshift(tag_for_slug("businesslink", "legacy_source"))
-      assert_equal @tag1, Slimmer::Artefact.new(@data).primary_section
-    end
-
-    it "should return nil if there are no sections" do
-      @data["tags"] = [tag_for_slug("businesslink", "legacy_source")]
-      assert_equal nil, Slimmer::Artefact.new(@data).primary_section
-    end
```

Comment:
These tests are still valid - they're testing that other tag types don't get returned by `primary_section`.  It might be worth  changing them to use a dummy tag type instead of this deprecated one.